packages/web/src/fixtures/dev-kit/get-policy-set-contract.ts

```diff
@@ -0,0 +1,8 @@
+import Web3 from 'web3'
+// import { createPolicySetContract } from '@devprtcl/dev-kit-js/esm/policy-set'
+// import { CustomOptions } from '@devprtcl/dev-kit-js/esm/option'
+
+// eslint-disable-next-line
+export const getPolicySetContract = (client: Web3, address?: string, options?: any) => {}
+// FIXME(@k3nt0w): this is test code for confirmation operation.
+// createPolicySetContract(client)(address, options)
```

Comment:
I couldn't get `PolicySetContract`... please give me some advice:bow:

Reply:
Thank you for the implementation!

To generate a PolicySet contract instance, you can use the familiar codes in "fixtures/dev-kit/client.ts" to create, and it is.

Like: https://github.com/dev-protocol/stakes.social/blob/270551f96abb332f37f47bf688bc59dc8d66f69c/packages/web/src/fixtures/dev-kit/client.ts#L16

So, for example:
```ts
client.policySet(await getContractAddress(client, 'policySet'))
```

Can generate that like this. :)

Reply:
I was completely wrong in my perception.... I seem to have been rather confused by what I read in dev-kit Interface:cry: Thank you for your advice with specific code!! I'll fix it soon.

Reply:
I'm sorry for the lack of documentation 🙇‍♂️ Many thanks!